Source/Meadow.Foundation.Core.Samples/Leds.RgbPwmLed_Onboard_Sample/MeadowApp.cs

```diff
@@ -18,48 +17,47 @@ public class MeadowApp : App<F7FeatherV2>
         public override Task Initialize()
         {
             Resolver.Log.Info("Creating peripherals...");
-            
+
             onboardLed = new RgbPwmLed(
                 Device.Pins.OnboardLedRed,
                 Device.Pins.OnboardLedGreen,
-                Device.Pins.OnboardLedBlue,
-                commonType: CommonType.CommonAnode);
+                Device.Pins.OnboardLedBlue);
 
             return Task.CompletedTask;
         }
 
         public override Task Run()
```

Comment:
You should be able to directly return TestColors - I think I did this when we were fighting to get it working